src/App.tsx

```diff
@@ -12,7 +12,11 @@ import './App.css';
 function App() {
   return (
     <>
+<<<<<<< HEAD
+      <NewContact />
+=======
     <Header/>
```

Comment:
You might want to add a whitespace before `/>` here or remove it other places.

Reply:
fixed it, thank you.